app/pages/project/project-page.jsx

```diff
@@ -111,7 +84,11 @@ export default class ProjectPage extends React.Component {
           user: this.props.user,
           workflow: this.props.workflow
         })}
-        {potentialFieldGuide}
+        {(this.props.location.pathname !== `/projects/${this.props.project.slug}`) &&
```

Comment:
This doesn’t check whether we’re actually on the home page. The `onHomePage` check from the old code is more reliable.

Reply:
Ok, good to know. It was confusing because that check was only being used with the navbar and I'm not sure if that worked or why the navbar was being added in both components.

Reply:
From memory, the string comparison misses trailing slashes but the router path check will catch those.

Reply:
There might be a better check where we look at `props.children` to see what type of component the project page is, but I'm not sure how that would work in practice.

Reply:
`React.Children.only(this.props.children).type.name === 'ProjectHomeContainer'` might work, using [Function.name](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/name) to get the only child's name.